sklearn/manifold/tests/test_spectral_embedding.py

```diff
@@ -258,7 +258,26 @@ def test_spectral_embedding_unnormalized():
     # Verify using manual computation with dense eigh
     laplacian, dd = graph_laplacian(sims, normed=False, return_diag=True)
     _, diffusion_map = eigh(laplacian)
-    embedding_2 = diffusion_map.T[:n_components] * dd
+    embedding_2 = diffusion_map.T[:n_components]
     embedding_2 = _deterministic_vector_sign_flip(embedding_2).T
 
     assert_array_almost_equal(embedding_1, embedding_2)
+
+
+def test_spectral_embedding_first_eigen_vector():
+    # Test that the first eigenvector of spectral_embedding
+    # is constant and that the second is not (for a connected graph)
+    random_state = np.random.RandomState(36)
+    data = random_state.randn(10, 30)
+    sims = rbf_kernel(data)
+    n_components = 2
+
+    for seed in range(10):
+        embedding = spectral_embedding(sims,
+                                       norm_laplacian=False,
+                                       n_components=n_components,
+                                       drop_first=False,
+                                       random_state=seed)
+
+        assert_array_almost_equal(np.diff(embedding[:, 0]), 0)
+        assert_array_equal(np.abs(np.diff(embedding[:, 1])) > 1e-3, True)
```

Comment:
Good point @vene  Thanks for your input. 
All I want is a test asserting that the elements are different (not constant).
The condition can be probably relaxed a bit with something like `assert_array_equal(np.std(embedding[:, 1]) > 1e-3, True)`.
Is this any better? What would you suggest instead of the current test?